session-02/exercise.js

```diff
@@ -3,7 +3,12 @@
   For example, for the input ["cat", "hat"], return ["CAT", "HAT"]
 */
 function transformArrayToUpper(listOfStrings) {
-  return;
+  for( let i=0; i< listOfStrings.length; i++)
+  {
+    listOfStrings = listOfStrings[i].toUpperCase();
```

Comment:
The error is on this line. This resets the entire input instead of a single element in the input array. You can correct it with `listOfStrings[i] = ...`